clickhouse/testcontainers/clickhouse/__init__.py

```diff
@@ -25,19 +27,24 @@ class ClickHouseContainer(DbContainer):
     ClickHouse database container.
 
     Example:
-
-        The example spins up a ClickHouse database and connects to it using the
-        :code:`clickhouse-driver`.
+        The example spins up a ClickHouse database and connects to it
+        using the :code:`clickhouse-driver`.
 
         .. doctest::
 
-            >>> import clickhouse_driver
             >>> from testcontainers.clickhouse import ClickHouseContainer
 
+            >>> import clickhouse_driver
             >>> with ClickHouseContainer("clickhouse/clickhouse-server:21.8") as clickhouse:
             ...     client = clickhouse_driver.Client.from_url(clickhouse.get_connection_url())
             ...     client.execute("select 'working'")
             [('working',)]
+
+            >>> import clickhouse_connect
+            >>> with ClickHouseContainer(port=8123) as clickhouse:
+            ...     client = clickhouse_connect.get_client(dsn=self.get_connection_url())
```

Comment:
there is no self in this context, it should be 
```suggestion
            ...     client = clickhouse_connect.get_client(dsn=clickhouse.get_connection_url())
```